db_scripts/mosip_pms/dml/pms-partner.csv

```diff
@@ -6,5 +6,5 @@ MOSIP.PROXY.SBI,mpolicygroup-default-mockdevice,MOSIP,headoffice9,9988776655,abc
 mpartner-default-resident,mpolicygroup-default-resident,IITB,mpartner-default-resident,9232121212,resident@mosip.io,,mpartner-default-resident,Credential_Partner,approved,TRUE,superadmin,now(),superadmin,now()
 mpartner-default-adjudication,mpolicygroup-default-adjudication,mpartner-default-adjudication,mpartner-default-adjudication,9232121212,adjudication@mosip.io,,mpartner-default-adjudication,Manual_Adjudication,approved,TRUE,superadmin,now(),superadmin,now()
 mpartner-default-cert,mpolicygroup-default-cert,mpartner-default-cert,mpartner-default-cert,9232121212,info@mosip.io,,mpartner-default-cert,Internal_Partner,approved,TRUE,superadmin,now(),superadmin,now()
-mpartner-default-digitalcard,mpolicygroup-deafult-digitalcard,mpartner-default-digitalcard,mpartner-default-digitalcard,9232121212,digitalcard@mosip.io,,mpartner-default-digitalcard,Credential_Partner,Activated,TRUE,superadmin,now(),superadmin,now()
-mpartner-default-mobile,mpolicygroup-default-mobile,residentapp-partner,mpartner-default-mobile,9232121212,info@mosip.io,,mpartner-default-mobile,Credential_Partner,approved,TRUE,superadmin,now(),superadmin,now()
\ No newline at end of file
+mpartner-default-digitalcard,mpolicygroup-deafult-digitalcard,IITB,mpartner-default-digitalcard,9232121212,digitalcard@mosip.io,,mpartner-default-digitalcard,Credential_Partner,Activated,TRUE,superadmin,now(),superadmin,now()
```

Comment:
@dhanendra06  ,the spelling of default is incorrect,kindly change the same.

Reply:
taken care